zebra-state/src/service.rs

```diff
@@ -1873,6 +1874,33 @@ impl Service<ReadRequest> for ReadStateService {
                 })
                 .wait_for_panics()
             }
+            ReadRequest::ValuePools(hash_or_height) => {
+                let state = self.clone();
+
+                tokio::task::spawn_blocking(move || {
+                    span.in_scope(move || {
+                        let block = state.non_finalized_state_receiver.with_watch_data(
+                            |non_finalized_state| {
+                                read::block(
+                                    non_finalized_state.best_chain(),
+                                    &state.db,
+                                    hash_or_height,
+                                )
+                            },
+                        );
+
+                        let value_pool_change = block
+                            .map(|b| b.chain_value_pool_change(&HashMap::new(), None))
```

Comment:
In `zcashd`, this seems to be the total pool value balances. Zebra currently only tracks those for non-finalized and finalized chain tips, so we may need to update the database format again to avoid reading up to half the block chain to compute these totals.